server/model/user.py

```diff
@@ -71,4 +77,14 @@ def delete_user(username):
     db.session.delete(user)
     db.session.commit()
 
-    return user
\ No newline at end of file
+    return user
+
+def add_completed_assignment(data):
+    if not does_user_exist(data['name']):
+        raise ResourceDoesNotExistError
+    
+    a = Assignment(name=a_name, user=u_name)
+    user = User.query.filter_by(name=u_name).first();
```

Comment:
Try to use the `get_user` function we wrote here, rather than essentially rewriting it yourself. It just makes for a little better encapsulation.